index.html

```diff
@@ -33,6 +33,10 @@
     <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-smooth.js"></script>
     <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-tizen.js"></script>
     <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/bitmovinplayer-ui.js"></script>
+     
+    <!-- Bitmovin Analytics -->
+     <script type="text/javascript" src="https://cdn.bitmovin.com/analytics/web/2/bitmovinanalytics.min.js"></script>
```

Comment:
Why don't we use the analytics player module (https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-analytics.js)? Any particular reason?

Reply:
it was the same in the other demo... not sure :D